data/campaigns/Descent_Into_Darkness/scenarios/10_Alone_at_Last.cfg

```diff
@@ -773,10 +941,35 @@
         [/endlevel]
     [/event]
 
+    [event]
+        name=alternate victory
```

Comment:
Shouldn't the alternate victory condition be added to the objectives?

Reply:
I like having hidden victory conditions for players who end up being good enough to win in situations where they're not strictly supposed to (here, killing Dela and the paladin).